README.md

```diff
@@ -20,10 +20,11 @@ Notes:
 
 * This adapter is a fork from the original [FEniCS-preCICE adapter](https://github.com/precice/fenics-adapter). Based on [v1.2.0](https://github.com/precice/fenics-adapter/releases/tag/v1.2.0).
 * This adapter is currently under development and experimental.
+* Target version: dolfinx v0.4.1
```

Comment:
Just to make sure: Is this the version you are using, as well?

Reply:
Is it possible to go for v0.5.2? I don't think there have been (m)any breaking changes, shouldn't be a big deal to update.

Reply:
On my local system I just installed `v0.5.2` via `apt`. I will see, if this causes any problems and fix them, if necessary.

Reply:
Everything works nicely with `v0.5.2` (ran tests + partitioned heat equation tutorial). I'm updating the target version now via https://github.com/precice/fenicsx-adapter/pull/20.